src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/behandlingsresultat/BehandlingsresultatEndringUtils.kt

```diff
@@ -170,17 +170,11 @@ private fun erEndringIBeløpForPersonOgType(
             val forrigeBeløp = forrige?.kalkulertUtbetalingsbeløp ?: 0
 
             if (erFremstiltKravForPerson) {
-                // Hvis det er søkt for person vil vi kun ha med endringer som går fra beløp > 0 til 0/null
-                when {
-                    forrigeBeløp > 0 && nåværendeBeløp == 0 -> true
-                    else -> false
-                }
+                // Hvis det er søkt for person vil vi kun ha med endringer som går fra beløp > 0 til 0/null siden vi skal ha innvilgelse når vi øker beløpet.
+                forrigeBeløp > 0 && nåværendeBeløp == 0
```

Comment:
Kan gå med på at vi ikke anser øking i beløp som en endring siden det er en innvilgelse, selv om jeg synes det er rart. 

Jeg er litt mer usikker på caset der vi går fra 0 til 50 prosent barnehageplass. Da vil beløpet bli mindre, men ikke null. Hvorfor skal ikke det anses som en endring?

Reply:
Det anses som en endring dersom det ikke er framstilt krav for personen (søknad el klage)

Reply:
Har nå snakket med Birgitte og virker som om vi vil ha at det skal være ENDRET sålenge det er forskjellig beløp. Mistenkes at det er legacy at vi spesialbehandler søknad og klage.

Reply:
Burde dette oppdateres til `forrigeBeløp != nåværendeBeløp`da?

Reply:
I så fall kan kommentaren fjernes